src/content/platform-overview/architecture/index.md

```diff
@@ -0,0 +1,152 @@
+---
+linkTitle: Platform architecture
```

Comment:
I have simplified this document to be readable. Before it has a long list of components and various information. My goal is to explain the Platform concept and how is the Architecture without going to much into details